.github/workflows/ci.yaml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  go:
+    name: Check index-generator build
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
nit: add `- name: Check out code` for this step is more clear and neat

Reply:
Sounds good 👍

Reply:
Updated